_posts/2019-08-26-how-to-use-gruntwork-service-catalog.adoc

```diff
@@ -1,14 +1,27 @@
 ---
-categories: Gruntwork
-image: /assets/img/guides/service-catalog/grunty-blocks.png
+categories: Foundations
+image: /assets/img/guides/service-catalog/grunty-grid.png
 excerpt: Learn about production-grade infrastructure, Terraform, Terragrunt, Packer, Docker, immutable infrastructure, versioning for infrastructure code, automated tests for infrastructure code, and more.
 tags: [aws, gcp, terraform, terragrunt]
+cloud: aws, gcp
```

Comment:
didn't know this would work! Why doesn't it have to be a list like the tags just above?

Reply:
Hm, good point. Don't know why it worked actually. Some YAML black magic? Anyway, fixed to proper syntax: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/169/commits/da5ecccdca2cc011e2156b432004b5642ebff2f8